app/controllers/users.js

```diff
@@ -41,3 +40,31 @@ exports.authenticate = (req, res, next) => {
     })
     .catch(reason => next(errors.defaultError(`Database error - ${reason}`)));
 };
+const hasSomething = param => param !== null && param !== '';
```

Comment:
this is equal to ask if (param) in javascript

Reply:
👍